main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapType.java

```diff
@@ -55,6 +55,13 @@ public UnifiedMapType(final Viewport viewport) {
         this.viewport = viewport;
     }
 
+    /** open map and scale to a given viewport */
+    public UnifiedMapType(final Viewport viewport, final String title) {
```

Comment:
With this new constructor in place, I assume that the constructor taking only a Viewport (not a name) is no longer used. Thus it should be removed.

Reply:
I introduced it for Wherigos... it was not there before. But as said, it is minor. No harm in leaving it in